src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        double area = Math.PI * Math.pow(radius, 2);
```

Comment:
Magic numbers should be avoided. Consider defining `2` as a constant with a clear name to explain what it represents, like `POWER_OF_RADIUS`.